src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Introduced `EnableResilientVMCreate` and `EnableResilientVMDelete` parameters to `Update-AzVmss` and `New-AzVmssConfig` cmdlets for enhanced VM resilience options.
```

Comment:
let's keep this consistent with other lines. 
Ex: * Added  parameter `-ResourceIdsOnly` to `Get-AzCapacityReservationGroup` cmdlet.

Reply:
This is AI generated, so I've marked it as improvements to make on ps cmd assistant.